src/components/DateRangePicker.js

```diff
@@ -367,13 +381,19 @@ class DateRangePicker extends React.Component {
                     />
                   </div>
                   <div style={styles.calendarWeek}>
-                    {['S', 'M', 'T', 'W', 'T', 'F', 'S'].map((day, i) => {
-                      return (
-                        <div key={day + i} style={styles.calendarWeekDay}>
-                          {day}
-                        </div>
-                      );
-                    })}
+                    {[{ label: 'S', value: 'Sunday' },
+                      { label: 'M', value: 'Monday' },
+                      { label: 'T', value: 'Tuesday' },
+                      { label: 'W', value: 'Wednesday' },
+                      { label: 'T', value: 'Thursday' },
+                      { label: 'F', value: 'Friday' },
+                      { label: 'S', value: 'Saturday' }].map((day) => {
```

Comment:
It is extremley important to give React keys to elements that are both consistent and unique.

Previously the solution was supplying keys that were unique but not consistent because it relied on the iteration index to create the key: `key={day + i}`.

In this case this is not currently causing problems, but it is one of my pet peeves and I wanted to fix it. I couldn't use the day value by itself because then it would be consistent but not unique.

So I created an array of objects with a label and a value. The value for each object is both consistent and unique, so it meets the constraints necessary to be used for a key by React to create iterable elements.